libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -46,7 +50,10 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
     private const val BUFFER_DILATION: Short = 1
     private const val TWO_LEGS: Short = 2
     private const val PRIMARY_ROUTE_INDEX = 0
+    private const val SINGLE_THREAD = 1
 
+    private val NavigatorDispatcher: CoroutineDispatcher =
```

Comment:
how are we going to mock that dispatcher in tests? maybe pass to the constructor?

Reply:
This is a good point. Still thinking how to add tests around that as we don't have tests for `MapboxNativeNavigatorImpl`. Any ideas?

Left 👇  unchecked intentionally 😅 

> - [ ] I have added tests that prove my fix is effective or that my feature works